packages/css-framework/src/components/_radio-card.scss

```diff
@@ -0,0 +1,42 @@
+@use "../abstracts/functions" as f;
+
+$animation-timing: 0.2s;
+$radio-focus-width: 0.1rem;
+
+.rn-radio-card {
+  display: inline-flex;
+  flex-direction: column;
+  width: 22rem;
+  padding: f.spacing("10");
+  margin: f.spacing("4");
+  border-radius: 4px;
+  border: 1px solid f.color("neutral", "100");
+  outline: none;
+
+  &:hover {
+    border-color: f.color("action", "400");
+    box-shadow: 0 0 0 3px f.color("action", "100");
+    cursor: pointer;
+
+    .rn-radio input ~ .rn-radio__checkmark {
+      border: 1px solid f.color("action", "500");
+      outline: none;
+      box-shadow: 0 0 0 $radio-focus-width f.color("action", "500");
+      transition: all $animation-timing;
+    }
+  }
+
+  &.is-selected {
+    border-color: f.color("action", "400");
+  }
+}
+
+.rn-radio-card__description {
+  color: f.color("neutral", "400");
+  padding-top: f.spacing("4");
+}
+
+.rn-radio-card .rn-radio__label {
+  font-size: f.font-size("l");
```

Comment:
Can we change this to `font-size("m")`? It's correct here based on the designs, however on reflection it looks better with a slightly reduced title size. I'll update the Sketch file to match too.